applications_tests/gls_navier_stokes/gls_droplet_marangoni_effect.prm

```diff
@@ -102,8 +103,9 @@ subsection physical properties
     subsection fluid-fluid interaction
       set first fluid id              = 0
       set second fluid id             = 1
-      set surface tension model       = constant
+      set surface tension model       = linear
       set surface tension coefficient = 0.023
+      set surface tension gradient    = 0.1
```

Comment:
I understand why you use the word gradient here,  but I ain't sure it's the right word...  but i can't find a better one...

Reply:
Maybe something like `surface_tension_temperature_gradient`, or `surface_tension_temperature_rate`?

Reply:
I'll change it to "temperature-driven surface tension gradient"